tests/web3js/eth_streaming_test.js

```diff
@@ -26,56 +26,63 @@ it('streaming of logs using filters', async() => {
     await new Promise((res, rej) => setTimeout(() => res(), 1000))
 
     // subscribe to new blocks being produced by bellow transaction submission
-    let blockCount = 0
-    let blockHashes = []
+    let blockTxHashes = []
+    let subBlocks = await ws.eth.subscribe('newBlockHeaders')
```

Comment:
why you think this is better?

Reply:
I took this pattern from `eth_streaming_filters_test`, which has never failed in CI. If we observe any problem, we can change it back.